deploy/charts/cert-manager/values.yaml

```diff
@@ -25,6 +25,8 @@ strategy: {}
   #   maxSurge: 0
   #   maxUnavailable: 1
 
+logLevel: 2
```

Comment:
Can we make this variable a `global`, and use the same setting across the `cainjector`, `controller` and `webhook` deployments? 😄

Reply:
webhook is currently hard coded to `--v=12` and floods my log collector with ~500k entries per day.

Reply:
Yep, definitely not ideal! If you can update this PR to make the variable a global, we should be able to hit two birds with one stone!